### ISSUE TITLE
Request:Addition of Hindi language

### ISSUE THREAD

**Issue by aghammadan** [OpenBMB/VoxCPM#47]
I want that we can clone english voice and genrate hindi audio.

**PLS DEVS**

**Comment by a710128** [comment]
in progress :D

**Comment by tensorjackal** [comment]
Any updates here?

**Comment by a710128** [comment]
> Any updates here?

Q1 2026, if everything goes as planned.

**Comment by darknight054** [comment]
@a710128 If I want to do LORA fine tuning for hindi, would you recommend transliterate the text or just normalizing and moving forward in dataset preprocessing? Also, would you recommend a full fine tuning? I read that @Ayin1412 could do a simple lora fine tuning for japanese language. Any help would be greatly appreciated

**Comment by Labmem-Zhouyx** [comment]
Hello, here are some suggestions :)
1. With limited data: Start with LoRA fine-tuning and consider transliterating the text to better leverage the existing vocabulary, since VoxCPM was not pretrained on Hindi characters.
2. With sufficient data (e.g., hundreds of hours): You may directly use Hindi script and perform full fine-tuning, or at least unfreeze the text embedding layer if using LoRA.

**Comment by darknight054** [comment]
@Labmem-Zhouyx Thanks a lot. Will try and let you know, I have good amount of data but then it would be expensive without trying the transliterated text pipeline. Will report back.

**Comment by darknight054** [comment]
@Labmem-Zhouyx 
I tried with roughly 32 hours of speech data on a single 12 gb gpu of hindi data and unfreezed the text embedding layer and got pretty promising results. Just in 4000 steps It can speak really nice, only some really tough letters or under appearing letters in the training data are kind of tough for it to speak but its pretty impressive. Thanks a lot for the guidance
